mindmeld/components/classifier.py

```diff
@@ -181,6 +179,10 @@ def fit(self, queries=None, label_set=None, incremental_timestamp=None, **kwargs
                 feature extractor function, or a callable which will be used as to
                 extract features.
 
+        Returns:
+            True if model was loaded and fit, False if a valid cached model exists.  The cached
+            model must be loaded if fit() returns False.
```

Comment:
What happens when valid cache model exists but the model is not loaded into mem?

Reply:
I'm not sure what you mean.  The training process will load this later via load()

We could load this, but we end up with extra work
1. load the cached model
2. unload the cached model when training the next one.
3. load the model again when nlp.build completes
4. dump the model again from the main process training it.

vs

1. verify a cached model exists
2. load it when nlp.build() finishes
3. dump it from the main process.